docs/zh/latest/plugins/csrf.md

```diff
@@ -29,11 +29,11 @@ title: csrf
 
 ## 属性
 
-| Name             | Type    | Requirement | Default | Valid | Description                                                  |
+| 名称             | 类型    | 必选项 | 默认值 | 有效值 | 描述                                                 |
 | ---------------- | ------- | ----------- | ------- | ----- | ------------------------------------------------------------ |
-|   name   |  string |    optional    | `apisix-csrf-token`  |    | 生成的 Cookie 中的 token 的名字，需要使用这个名字在请求头携带 Cookie 中的内容 |
-| expires |  number | optional | `7200` | | CSRF Cookie 的过期时间(秒) |
-| key | string | required |  |  | 加密 token 的秘钥 |
+|   name   |  string |    可选    | `apisix-csrf-token`  |    | 生成的 Cookie 中的 token 的名字，需要使用这个名字在请求头携带 Cookie 中的内容 |
+| expires |  number | 可选 | `7200` | | CSRF Cookie 的过期时间(秒) |
+| key | string | 必选 |  |  | 加密 token 的秘钥 |
```

Comment:
```suggestion
|   name   |  string |    否    | `apisix-csrf-token`  |    | 生成的 Cookie 中的 token 的名字，需要使用这个名字在请求头携带 Cookie 中的内容 |
| expires |  number | 否 | `7200` | | CSRF Cookie 的过期时间(秒) |
| key | string | 是 |  |  | 加密 token 的秘钥 |
```

Reply:
I think it should be changed to “是” or “否”. This is more suitable for "必选项".

Reply:
@juzhiyuan what you think?  if we changed to "是" or "否"，we should changed all docs includ en&zh

Reply:
I just checked Stripe and GitHub's API References, they're different from Apache APISIX API References.

I have two styles:

1. Required (必填) -> Yes, No
2. Optional (可选) -> Required, Optional

After consideration, how about using `Required` as column name? also cc @navendu-pottekkat

Reply:
If we do, I'll change it all on this PR

Reply:
> translate some zh/*.md table properties to chinese

Please wait for @navendu-pottekkat, @kwanhur, and @hf400159's confirmation, because if we change Chinese, we should also update English version :)

Reply:
I would suggest Required -> Yes/No

Reply:
No, I don't we should change them on this PR, before the document specification comes out, I prefer that keep consistent format.

Reply:
No, I don't we should change them on this PR, before the document specification comes out, I prefer that keep consistent format.

Reply:
> No, I don't we should change them on this PR, before the document specification comes out, I prefer that keep consistent format.

yes;  I aggree with you

Reply:
I agree. And for the specification we can follow this I would suggest Required -> Yes/No.

Reply:
OK!